### PR TITLE
Fix shellcheck errors in pki

### PR DESCRIPTION
**Pull request by ckelleyRH**
The directive to fail on error is still present, so the job now succeeds.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -856,7 +856,7 @@ if [ "$WITHOUT_TEST" = true ] ; then
 fi
 
 if [ "$DEBUG" = true ] ; then
-    echo "rpmbuild -bs ${OPTIONS[@]} $SPEC_FILE"
+    echo "rpmbuild -bs " + "${OPTIONS[@]}" + "$SPEC_FILE"
```
Comment (fmarco76) [inline build.sh:859]: Actually, I have always used the syntax above in shell code. For me is more readable but the new version is OK

**Review by fmarco76 — APPROVED** [review]
LGTM

**Comment by ckelleyRH** [comment]
Cheers @fmarco76 !

File: build.sh (hunk 1 above)
Comment (edewata) [reply build.sh]: Do we need the `+` sign? Might be cleaner without it (but keep the new quotes).

Comment (ckelleyRH) [reply build.sh]: Ah yeah good point, I'll push that optimisation directly, thanks!

**Pull request merged**